lib/interface/cli/commands/team/create.cmd.js

```diff
@@ -0,0 +1,70 @@
+const Command = require('../../Command');
+const { team } = require('../../../../logic').api;
+const createRoot = require('../root/create.cmd');
+const fs = require('fs');
+
+
+const command = new Command({
+    command: 'teams <name>',
+    aliases: ['team', 'tm'],
+    parent: createRoot,
+    description: 'Create a team',
+    usage: 'You can create a new team specifying only name of team or specify path to json file for include extended options',
+    webDocs: {
+        category: 'Teams',
+        title: 'Create team',
+    },
+    builder: (yargs) => {
+        return yargs
+            .positional('name', {
+                describe: 'Name of team',
+            })
+            .option('team-file', {
```

Comment:
there is already a general option to create resource from file , with -f option , there is no need to add a specific option 'team-file' , you can take a look on the other resources with create option